app/handlers.go

```diff
@@ -217,12 +224,30 @@ func ReplaceArticle(env *Env, w http.ResponseWriter, r *http.Request) error {
 	newPost.Date = env.Clock.Now()
 
 	// Make call to rethink database and get changes back
-	resp, err = db.DB("content").Table("posts").Get(articleId, db.InsertOpts{ReturnChanges: true}).Replace(newPost).RunWrite(env.Session)
+	resp, err = db.DB("content").Table("posts").Get(articleId).Replace(newPost, db.ReplaceOpts{ReturnChanges: true}).RunWrite(env.Session)
 	if err != nil {
 		fmt.Print(err)
 		return StatusError{500, err}
 	}
 
+	// Coming through as interface{}
+	newAlbumID := getPostDataFromRethinkChanges(resp.Changes[0].NewValue)
+	oldAlbumID := getPostDataFromRethinkChanges(resp.Changes[0].OldValue)
```

Comment:
Is there a guarantee that the resp.Changes array will always have a value? If not, then we need to check whether it exists